contrib/mypy/src/python/pants/contrib/mypy/tasks/mypy_task.py

```diff
@@ -34,8 +55,12 @@ def prepare(cls, options, round_manager):
   @classmethod
   def register_options(cls, register):
     register('--mypy-version', default='0.710', help='The version of mypy to use.')
-    register('--config-file', default=None,
+    register('--config-file', default='build-support/mypy/mypy.ini',
```

Comment:
Bad change. This should default to None still. We don't expect every Pants user to have a file at `build-support/mypy/mypy.ini`. This is how Pants itself happens to do it, but should only be configured in our local `pants.ini` - not in the default settings that get used by everyone.